### DISCUSSION TITLE
Quarto pdf rendering error "compilation failed- error"

### DISCUSSION THREAD

**Discussion by em9898** [quarto-dev/quarto-cli#4865] [Q&A]
I've been using Quarto on rstudio over the last few months, rendering quarto files to pdf without any issues. 

Recently, I am unable to render my quarto files to pdf and I get the following error. 

compilation failed- error
Text line contains an invalid character.
l.218 i Use the ^^[
                   ]8;;http://conflicted.r-lib.org/^^Gconflicted package^^[]... 

How can I fix this?

**Comment by guaxen** [comment]
Hi, how did you manage to resolve this issue? I have exactly the same error. Thanks!

**Reply by em9898** [reply]
For me, the problem was when I load packages in blocks of code that has a lengthy message, it failed to render. I solved this by adding "include = FALSE" in the ```{r include = FALSE} part of code where I load libraries. I found this page helped https://yihui.org/tinytex/r/#debugging

**Reply by guaxen** [reply]
Thank you, this worked for me as well!

**Comment by cscheid** [comment]
Thanks for the report. We need a minimal reproducible example before we can go ahead and diagnose the issue. Can you provide one? Thanks!
